### PR TITLE
Change base images to the upstream ones

### PR DESCRIPTION
**Pull request by averevki**

### CODE REVIEW

File: base/go-httpbin/deployment-config.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
               value: /tls/tls.crt
             - name: HTTPS_KEY_FILE
               value: /tls/tls.key
-          image: quay.io/rh_integration/go-httpbin:smadis
+          image: docker.io/jsmadis/go-httpbin:latest
```
Comment (mganisin) [inline base/go-httpbin/deployment-config.yaml:34]: please keep this unchanged, it will be new upstream

File: base/httpbin/deployment.yaml [hunk 2]
```diff
@@ -14,7 +14,7 @@ spec:
     spec:
       containers:
         - name: httpbin
-          image: quay.io/rh_integration/httpbin:smadis
+          image: docker.io/jsmadis/httpbin:latest
```
Comment (mganisin) [inline base/httpbin/deployment.yaml:17]: please keep this unchanged, it will be new upstream

File: overlays/template/kustomization.yaml [hunk 3]
```diff
@@ -3,10 +3,10 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 images:
-  - name: quay.io/rh_integration/httpbin
+  - name: docker.io/jsmadis/httpbin
```
Comment (mganisin) [inline overlays/template/kustomization.yaml:6]: please keep this unchanged, it will be new upstream

File: overlays/template/kustomization.yaml [hunk 4]
```diff
@@ -3,10 +3,10 @@ apiVersion: kustomize.config.k8s.io/v1beta1
...
     newName: quay.io/rh_integration/httpbin
     newTag: smadis
-  - name: quay.io/rh_integration/go-httpbin
+  - name: docker.io/jsmadis/go-httpbin
```
Comment (mganisin) [inline overlays/template/kustomization.yaml:9]: please keep this unchanged, it will be new upstream

File: base/jaeger/deployment.yaml [hunk 5]
```diff
@@ -28,7 +28,7 @@ spec:
               value: "9411"
             - name: MAX_TRACES
               value: "50000"
-          image: quay.io/rh_integration/jaeger-aio:latest
+          image: docker.io/jaegertracing/all-in-one:latest
```
Comment (pehala) [inline base/jaeger/deployment.yaml:31]: If we want to stay off the docker.io as much as possible, we can also use `quay.io/jaegertracing/all-in-one`, thoughts @mganisin @ficap ?

Comment (ficap) [reply base/jaeger/deployment.yaml:31]: Let's go for it, then

**Pull request merged**